common/config/ringpop.go

```diff
@@ -121,6 +123,8 @@ func parseBootstrapMode(
 		return BootstrapModeCustom, nil
 	case "dns":
 		return BootstrapModeDNS, nil
+	case "dns-srv":
```

Comment:
can you help update this comment: https://github.com/uber/cadence/blob/8510816511e1532514bd11aadc09966278db3975/common/config/config.go#L160

to 
`BootstrapMode is a enum that defines the ringpop bootstrap method, currently supports: hosts, file, custom, dns and dns-srv`